components/chat/Chat.tsx

```diff
@@ -26,28 +26,54 @@ export default function Chat() {
 
   const db = useDB()
 
+  //ensure each notification only shows up once
+  const notifInit = useRef(false)
```

Comment:
The errors only show up twice on development builds - its because toast() is called twice, once by the nextJS server and once by the frontend. Once you build the app, we no longer have a NextJS server running, so they only show up once.

Good idea though.